src/coreComponents/common/StdContainerWrappers.hpp

```diff
@@ -40,6 +72,10 @@ class StdVectorWrapper : public std::vector< T, Allocator >
   {
     if constexpr (USE_STD_CONTAINER_BOUNDS_CHECKING)
     {
+      if( index >= this->size() )
+      {
+        std::cout<<"noooooooooo"<<std::endl;
+      }
```

Comment:
```suggestion
```

Reply:
oh...yeah...Instead of relying on `std::vector::at()`,  I should just throw a `GEOS_ERROR` here so we 1) get nice stack trace, 2) can set a breakpoint....otherwise the stack trace is misleading and there is nowhere to set the breakpoint.